scripts/db-refresh.js

```diff
@@ -0,0 +1,72 @@
+const { execSync } = require('child_process');
+const fs = require('fs');
+
+// check the environment variable is set
+const mongodb_uri = process.env.MONGODB_URI;
+if (mongodb_uri === undefined) {
+    const localhost_uri_example = 'mongodb://localhost/5e-database';
+    console.error(
+        'MONGODB_URI must be defined before running the script:'
+        + '\nOn Apple or Linux:'
+        + `\n  $MONGODB_URI=${localhost_uri_example} npm run db:refresh\n`
+        + '\nOn Windows:'
+        + '\n   PowerShell:'
+        + `\n       $env:MONGODB_URI="${localhost_uri_example}"; npm run db:refresh`
+        + '\n   CMD prompt:'
+        + `\n       set MONGODB_URI=${localhost_uri_example} && npm run db:refresh\n`);
+    process.exit(1);
+}
+
+// check mongoimport can be found and executed
+try {
+    execSync('mongoimport --version');
+} catch (e) {
+    console.error('could not execute mongoimport - make sure the directory containing mongoimport is visible in your $PATH environment variable')
+    process.exit(1);
+}
+
+const json_db_dir = 'src';
+const json_db_collection_prefix = '5e-SRD-';
+const json_data_pattern = `\\b${json_db_collection_prefix}(.+)\.json\\b`;
+
+const regex = new RegExp(json_data_pattern);
```

Comment:
This will end up picking up the Test file. Though in all honesty, I'm not even sure why that file exists.

Reply:
I might just remove the file, honestly.

Reply:
I made a separate PR to fix this: https://github.com/bagelbits/5e-database/pull/227

Reply:
I edited the PR description text to remove the mention of the *test* JSON file + collection.